packages/commcare/test/index.js

```diff
@@ -216,19 +214,21 @@ describe('getCases', () => {
     };
 
     const { data } = await execute(get('case/12345'))(state);
-
-    expect(data.case_id).to.equal('12345');
-    expect(data.properties.case_type).to.equal('pregnancy');
+    expect(data[0].case_id).to.equal('12345');
+    expect(data[0].properties.case_type).to.equal('pregnancy');
   });
 
   it('should fetch cases and call the callback', async () => {
     testServer
       .intercept({
-        path: `/a/${domain}/api/v0.5/case`,
+        path: /\/a\/my-domain\/api\/v0\.5\/case/,
```

Comment:
I don't think we need a regex path AND a query object. Ideally path and query would be fine.

Tbh I still think the undici mock should match a path and ignore query parameters. I think we need to investigate that a little further.